book/src/user/shielded-scan-grpc-server.md

```diff
@@ -0,0 +1,143 @@
+# Zebra Shielded Scanning gRPC Server
+
+## Get Started
+
+### Setup
+
+After setting up Zebra Shielded Scanning, add a `listen_addr` field to the shielded-scan configuration:
```

Comment:
```suggestion
After setting up [Zebra Shielded Scanning](shielded-scan.md), add a `listen_addr` field to the shielded-scan configuration:
```

Reply:
Thank you, applied in https://github.com/ZcashFoundation/zebra/pull/8288/commits/6a1057ae5418a1afebdf89d895f5f3516cd6466f.